github/pulls.go

```diff
@@ -134,6 +135,29 @@ func (s *PullRequestsService) Get(owner string, repo string, number int) (*PullR
 	return pull, resp, err
 }
 
+// Get pull request raw format (diff or patch)
+func (s *PullRequestsService) GetRaw(owner string, repo string, number int, opt *MediaRawTypeOption) (string, *Response, error) {
+	u := fmt.Sprintf("repos/%v/%v/pulls/%d", owner, repo, number)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return "", nil, err
+	}
+
+	if opt.Type == Diff {
+		req.Header.Set("Accept", mediaTypeDiff)
+	} else if opt.Type == Patch {
+		req.Header.Set("Accept", mediaTypePatch)
+	}
+
+	ret := new(bytes.Buffer)
+	resp, err := s.client.Do(req, ret)
+	if err != nil {
+		return "", resp, err
+	}
+
+	return string(ret.Bytes()), resp, err
```

Comment:
`string(ret.Bytes())` can be simplified to `ret.String()`. But are you sure that returning a `string` rather than `[]byte` makes the most sense?